hopsanc/src/hopsanc.cpp

```diff
@@ -0,0 +1,275 @@
+#include "hopsanc.h"
+#include <iostream>
+
+#include "HopsanCore.h"
+#include "HopsanEssentials.h"
+#include "ComponentSystem.h"
+
+static hopsan::ComponentSystem *spCoreComponentSystem = nullptr;
+static hopsan::HopsanEssentials gHopsanCore;
+
+static double startTime, stopTime;
+
+//! @brief Prints all waiting messages
+//! @param[in] printDebug Should debug messages also be printed
+void printWaitingMessages(hopsan::HopsanEssentials& hopsanCore, bool printDebug, bool silent)
+{
+    if(silent) return;
+
+    hopsan::HString msg, type, tag;
+    while (hopsanCore.checkMessage() > 0) {
+        hopsanCore.getMessage(msg,type,tag);
+        if (type == "debug") {
+            if (printDebug) {
+                std::cout << msg.c_str() << "\n";
+            }
+        }
+        else {
+            std::cout << msg.c_str() << "\n";
+        }
+    }
+}
+
+
+int loadModel(const char* path) {
+    spCoreComponentSystem = gHopsanCore.loadHMFModelFile(path, startTime, stopTime);
```

Comment:
If a model has been previously loaded, it should be deleted before loading a new one